assets/textures/icon_item_static/icon_item_static.c

```diff
@@ -710,16 +710,16 @@ u64 gCBtnSymbolsTex[TEX_LEN(u64, gCBtnSymbolsTex_WIDTH, gCBtnSymbolsTex_HEIGHT,
 #include "assets/textures/icon_item_static/gCBtnSymbolsTex.ia8.inc.c"
 };
 
-#define gNamePanelLeftTex_WIDTH 72
-#define gNamePanelLeftTex_HEIGHT 24
-u64 gNamePanelLeftTex[TEX_LEN(u64, gNamePanelLeftTex_WIDTH, gNamePanelLeftTex_HEIGHT, 8)] = {
-#include "assets/textures/icon_item_static/gNamePanelLeftTex.ia8.inc.c"
+#define gInfoPanelBgLeftTex_WIDTH 72
+#define gInfoPanelBgLeftTex_HEIGHT 24
+u64 gInfoPanelBgLeftTex[TEX_LEN(u64, gInfoPanelBgLeftTex_WIDTH, gInfoPanelBgLeftTex_HEIGHT, 8)] = {
+#include "assets/textures/icon_item_static/gInfoPanelBgLeftTex.ia8.inc.c"
 };
```

Comment:
theres more stuff to change, right?

Reply:
You mean this should be renamed to "info plate" ?

Reply:
werent you calling the Z and R buttons "info panel left" and "info panel right" before? but thats changed now?